### PR TITLE
fix: renderings title

### PR DESCRIPTION
**Pull request by jaydeepsingh25**
resolves https://github.com/Shared-Reality-Lab/IMAGE-browser/issues/348.

### CODE REVIEW

**Comment by Cybernide** [comment]
I would say 'yes' it's grammatical, but I'm giving this to @JRegimbal and @emmanuelwilson because I'm not certain if this follows convention in so far as UI elements go.

**Comment by emmanuelwilson** [comment]
The very last one is still in English, I don't know if that was intentional or not. Giving a quick read over and I think it looks good! I am personally not a fan of the use of "Rendu" for rendering as it doesn't feel right but I don't have any better suggestions. I will ask one of our other french members and get back to you guys! I would want to suggest "interpretation" instead but that isn't precisely the right word. It is also possible that I am just not used to "rendu" being used in this context.

**Comment by JRegimbal** [comment]
It seems « rendu » would be the preferred translation, at least [it is for the OQLF](https://vitrinelinguistique.oqlf.gouv.qc.ca/fiche-gdt/fiche/2076095/rendu). That being said, I thought we were moving away from showing "rendering" to the user in the extension, preferring either "representation" or "interpretation"? We do use this terminology inconsistently though (e.g., interpretation/interprétation for the labels but then full rendering/rendu complet in segmented audio).

tl;dr I think the translation is probably fine, but the title may not be following our own conventions in both cases.

**Comment by jeffbl** [comment]
Assigned to @Cybernide for wording heads-up/decision.

**Comment by jeffbl** [comment]
@Cybernide Sorry, was not assigned to me and missed notification on this. If something is blocked on me, please assign to me. Would rather just do it all in one shot if not too much effort to sync up language to be consistent within and across languages, but if that will take significant time, ok with doing this as first step. Just make sure to log the new work item and link it here.
